src/openforms/submissions/api/serializers.py

```diff
@@ -148,6 +148,14 @@ class SubmissionSerializer(serializers.HyperlinkedModelSerializer):
         read_only=True,
     )
 
+    anonymous = serializers.BooleanField(
+        label=_("Anonymous"),
+        help_text=_("Whether the submission was started anonymously or not."),
+        required=False,
+        write_only=True,
+        default=False,  # type: ignore
```

Comment:
I think this is an issue with django and typing, but at this point I don't know if I can trust my type checker :stuck_out_tongue: 
![Screenshot from 2024-05-23 15-23-06](https://github.com/open-formulieren/open-forms/assets/29249171/100b0606-8a2e-4933-995d-289ea4994181)

Reply:
yeah it is, maybe that the drf-stubs help with this, but otherwise you can indeed just `type: ignore` it :)